packages/blade/src/components/Badge/badgeTokens.ts

```diff
@@ -0,0 +1,12 @@
+import type { BadgeProps } from './Badge';
+
+type BadgeMinHeight = 20 | 24;
+
+const minHeight: Record<NonNullable<BadgeProps['size']>, BadgeMinHeight> = {
+  small: 20,
+  medium: 24,
+};
+
+const maxWidth = 100;
```

Comment:
Waiting for @saurav12 to confirm `maxWidth`

Reply:
Please check for `maxWidth` for each device on [Platform](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A157) & [Characteristics](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8110%3A50) section on Figma.